### PR TITLE
Exercise 13

### PR DESCRIPTION
**Pull request by charanraj-thiyagarajan-cdw-com**

### CODE REVIEW

File: Exercise-13/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;500&display=swap" rel="stylesheet" />
+
+    <!-- external css -->
+    <link rel="stylesheet" href="assets/css/styles.css" />
+  </head>
+  <body>
+    <!-- header section start -->
+    <header class="header-content">
+      <span>FRIENDS</span>
```
Comment (DurgaBNVL) [inline Exercise-13/index.html]: could you let me know if this could be heading tag instead of span ?

File: Exercise-13/js/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import data from "../assets/friends.json" assert { type: "json" };
+
+const cardsContainer = document.getElementById("cardsContainer");
+
+function addFriendsData() {
+  const result = new DocumentFragment();
```
Comment (DurgaBNVL) [inline Exercise-13/js/index.js]: could you use proper naming conventions instead of generic ones like result

File: Exercise-13/js/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import data from "../assets/friends.json" assert { type: "json" };
+
+const cardsContainer = document.getElementById("cardsContainer");
+
+function addFriendsData() {
```
Comment (DurgaBNVL) [inline Exercise-13/js/index.js:8]: could you also add comments to functions

File: Exercise-13/js/index.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    image.setAttribute("alt", friendDetail.first_name);
+    friendImageContainer.appendChild(image);
+
+    // detail container
+    const friendDetailsContainer = document.createElement("div");
+    friendDetailsContainer.classList.add("friend-details");
+    const friendName = document.createElement("p");
+    friendName.classList.add("friend-name");
+    friendName.innerText = friendDetail.first_name + " " + friendDetail.last_name;
+    const friendEmail = document.createElement("p");
```
Comment (DurgaBNVL) [inline Exercise-13/js/index.js]: could you confirm if email also could be in p tag ?

File: Exercise-13/index.html (hunk 1 above)
Comment (charanraj-thiyagarajan-cdw-com) [reply Exercise-13/index.html]: Yes, I have seen that it is best to use heading tags for best seo practices. I have changed it to h1 tag

File: Exercise-13/js/index.js (hunk 2 above)
Comment (charanraj-thiyagarajan-cdw-com) [reply Exercise-13/js/index.js]: Ok, I will do that

File: Exercise-13/js/index.js (hunk 3 above)
Comment (charanraj-thiyagarajan-cdw-com) [reply Exercise-13/js/index.js:8]: ok, I will do that

File: Exercise-13/js/index.js (hunk 4 above)
Comment (charanraj-thiyagarajan-cdw-com) [reply Exercise-13/js/index.js]: For emails, it is best to use anchor tag. I have changed it.

**Pull request merged**